pynvml/nvml.py

```diff
@@ -326,9 +337,9 @@
 NVML_TOPOLOGY_SYSTEM = 50
 
 _nvmlGpuP2PCapsIndex_t = c_uint
-NVML_P2P_CAPS_INDEX_READ = 0
+NVML_P2P_CAPS_INDEX_READ = 0,
```

Comment:
```suggestion
NVML_P2P_CAPS_INDEX_READ = 0
```
This seems like it's probably a bug? Returning a `tuple` instead of an `int` would probably be an error here.

Reply:
I'm now seeing that the types with these comma typos aren't "used" anyway, so this isn't really a high priority. Fine with me to copy in the nvidia-ml-py code "as is".